### PR TITLE
chore: make Keycloak theme compatible with current Keycloak version

### PR DESCRIPTION
**Pull request by ciyer**
Use updated Keycloak theme

/deploy

### CODE REVIEW

**Comment by ableuler** [comment]
The ci deployment does not have an identity provider as login option configured. I guess you tested this on a personal dev deployment with an identity provider set up?

**Comment by rokroskar** [comment]
What is this bit at the bottom? I looked at it with Firefox and Chrome, looks the same 
image

**Comment by lorenzo-cavazzi** [comment]
Same thing here

**Comment by ciyer** [comment]
That's the background that is included with the new Keycloak theme. Sorry didn't catch that since my screen isn't large enough to see that, but I will make a fix.

**Comment by ciyer** [comment]
Ok, fixed in https://github.com/SwissDataScienceCenter/keycloak-theme/pull/10

You can see it in action if you try to login on this instance: https://sekhar.dev.renku.ch/

File: helm-chart/renku/values.yaml [hunk 1]
```diff
@@ -201,7 +201,7 @@ keycloak:
 
   extraInitContainers: |
     - name: theme-provider
-      image: renku/keycloak-theme:2.0.1
+      image: renku/keycloak-theme:3.0.0
```
Comment (rokroskar) [inline helm-chart/renku/values.yaml:204]: Lets use a new tag, `3.0.1`?

**Review by rokroskar — CHANGES_REQUESTED** [review]

**Comment by ciyer** [comment]
I have activated two dummy identity providers on the PR deployment. You will not be able to log into RenkuLab using them, but you can see what the buttons look like.

**Review by ableuler — APPROVED** [review]
LGTM!

**Review by rokroskar — APPROVED** [review]

**Pull request merged**